src/main/java/com/bankofspring/hello/HelloWorldImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.bankofspring.hello;
+
+public class HelloWorldImpl implements GreetingService {
+
+	private String greeting = "Hello, world!";
+	
+	public HelloWorldImpl() {
+		super();
```

Comment:
True. I was playing around with the constructors and ended up leaving this in for clarity.